### DISCUSSION TITLE
Custom RF Profiles !

### DISCUSSION THREAD

**Discussion by al-x753** [ExpressLRS/ExpressLRS#2887] [Ideas]
Let me know if this feature already exists but as far as I'm aware it isn't : 
I would love to create Custom RF Profiles or Presets depending on what aircraft/flight style I'm using my TX Module with.

Typical example with my 915Mhz module : 

- For my 7" Long Range/Mountain surfing rigs I max out the packet rate for low latency and blast relatively high power for maximum penetration and signal reliability.
- For my Gliders and FPV Wings I lower the packet rate because I don't need low latency and I'm going much further, and I use dynamic power to drain less battery as I can fly for multiple hours on one pack.

Switching between theses 2 currently forces me to go through all of theses settings one by one where it would be much more convenient to switch between custom presets : Quadcopter Mountain Surfing and Long Range Fixed Wing.

This example only covers 2 use cases but I believe it can be very useful for 2.4Ghz as well with many other variables : Gemini mode, Telemetry Toggle etc. kinda like Ghost does it : 

![image](https://github.com/user-attachments/assets/db90bd52-a334-459f-a516-88c46552275d)

To push it even further we could even assign and save that RF Profile on the Receiver itself, which would allow pilots to not even have to manually switch profile on the TX when switching between multiple crafts intended for different use cases. Just plug and play, all set up and ready to go.

Does that makes sense ?

**Comment by pkendall64** [comment]
We already have different configurations in ExpressLRS that differ by the CRSF receiver number.
So you can set a different receiver number for different models in EdgeTX and the settings in the TX module will be different for each receiver number. 
See https://www.expresslrs.org/software/model-config-match/?h=mode#model-configuration

**Reply by al-x753** [reply]
That's pretty cool, I will definitely use it, thanks for your answer.

I currently set all my planes up on the same EdgeTX model per transmitter, for example ELRS 2.4 and ELRS 0.9, that way I basically never have to change between model when I'm exclusively flying on one frequency.

What I was thinking of would allow to automatically change RF Profiles without creating an EdgeTX Model for each RF Profile, it would be slightly more convenient imo.
